remini/misc/premium/annotations/UnlockPremiumCompatibility.kt

```diff
@@ -0,0 +1,7 @@
+package app.revanced.patches.remini.misc.premium.annotations
+
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Package
+
+@Compatibility([Package("com.bigwinepot.nwdn.international", ["3.7.46.202160002"])])
```

Comment:
This will fail to work vis the SemanticVersion parser in this repository. Adjust it so it works.